FastTimeSeries/src/models.jl

```diff
@@ -0,0 +1,27 @@
+
+"""
+blockmodel(inblock::TimeSeriesRow, outblock::OneHotTensor{0}, backbone)
+
+Construct a model for time-series classification.
+"""
+function blockmodel(inblock::TimeSeriesRow,
+                outblock::OneHotTensor{0}, 
+                backbone)
+    data   = rand(Float32, inblock.nfeatures, 32, inblock.obslength)
+    # data   = [rand(Float32, inblock.nfeatures, 32) for _ ∈ 1:inblock.obslength]
+    output = backbone(data)
```

Comment:
[`Flux.outputsize`](https://fluxml.ai/Flux.jl/stable/utilities/#Flux.outputsize) was designed for precisely this.